[Review/Test merge Ready]Adds Voxys yaya

Ziiro opened the pull request:
:cl:
add: Vox as roundstart spiecies
/:cl:

* Vox in General: Stunned for slightly longer (unbalanced in human gravities). Higher tox damage. Space  *resistant*. Needs higher tech to clone.

![dreamseeker_2017-12-22_13-36-28](https://user-images.githubusercontent.com/26494679/34311933-db28a126-e71d-11e7-8ceb-6cfe08f1d155.png)

Let's get this started.
  
"Auroras" skin by Prometh
Vox language icon by Xantholne
Clothing sprites and code taken from/inspired by Paradise
Ling armor & Space suit done by Vivi

Ziiro committed:
Initial commit. A long way to go.

EldritchSigma commented:
Yaya is goods. Is makings Ryleigh Youngs scareds agains!

Ziiro committed:
okay this is going to be harder than I thought
-Bodies work
-Quills Work
-Breath Mask works.. mostly.
-A lot of the groundwork is now in place

Ziiro commented:
Straight up 🅱️roke:
* Vox Facial hair doesn't work. Probably made a typo somewhere
* Prolitus don't have tails. How the fuck does tail code *even work.* How am I going to get tail code to work with the body code. **AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA**
* Their breathing tank pressure is.. too low? They don't get enough to breathe on spawn, but if you crank it up to 25KPA, they're fine. ???????
* They are always choking on oxygen, despite wearing a mask. ??????
* Organs are missing icons
* Major mechanics are not implemented
* Need unique, colorable eyes for both Prolitus (two eyes) and Auroras (three eyes)

Ziiro committed:
FIXES FACIAL ACCESSORIES

Ziiro commented:
Prolitus on the Left, Auroras on the Right.

![dreamseeker_2017-12-22_20-50-19](https://user-images.githubusercontent.com/26494679/34316739-d8152aa4-e759-11e7-92a7-c67b37a1d3d9.png)
![dreamseeker_2017-12-22_20-50-32](https://user-images.githubusercontent.com/26494679/34316740-da526052-e759-11e7-99f3-fb71d45e4848.png)
![dreamseeker_2017-12-22_20-50-50](https://user-images.githubusercontent.com/26494679/34316741-dbbc5ea2-e759-11e7-8159-19643428de41.png)

AndrewMontagne committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

SmokingKilz commented:
No Vox that speak normally, please. That really, really breaks the whole idea of Vox for me and really removes a rather unique part of their identity.

EldritchSigma commented:
The shitbird caste which probably most will play will still talk like... like they talk on all other servers yaya

SmokingKilz commented:
Even if it was a 99% to 1% difference I think it would really, really mess with how Vox are meant to be, my personal opinion. I wouldn't want to see perfectly speaking Vox, I just thing it's cringy.

EldritchSigma commented:
I mean my voxxy will always be a shitbird who cannot be cloned. I will always let "her" talk like a retard and be a cute shitty space bird who hides in a locker when afraid and screams when seing skulls and skeletons... I found Vox on Para not talking like Vox really triggering aswell. And Vox who go on Quad-Esword Murdersprees. Rachidi is the only character of mine where I have restrains towards killing witnesses in this game. Else I just go the most efficient path. I love Vox (figurativly).

Spartan6 commented:
Vox talking normally seriously takes me out of the moment. It’s not normal.

I’m fine with them being differently balanced but they need to be yaya birds okie? Good.

Birdtalon commented:
Skrek 👍

Ziiro commented:
Okay, scrapping the two vox types because it's a coding hurdle I don't want to deal with.

The three eye/super alien look will simply be a body choice.

Ziiro commented:
Oh hey, I fixed the breathing problem. kikiki.

Rework:

There is now only "Vox outcast." Of those outcast, you can pick between the two-eyed normal vox, or the fancy 3-eyed "Auroras" vox. The current plan is to have the tail and eyes read the body, and then apply their feature based on that.

Lore wise: I think I'd still like to allow vox to speak normally if they wish.

Current issues:
* I don't think toxmod is affecting the damage number for breathing o2.
* Organs are missing icons
* Skinning a vox doesn't have an icon
* Quill (head and face) styles aren't greyscale
* Default and Random features don't work
* No Namelist
* Tails and Eyes don't attach properly, but I've got the concept/structures in place.

Ziiro committed:
Out with castes, in with.. Outcasts?
-No more Auroras/Prolitus. Just vox outcast.
-Prototype code for eye/tail selection
-Removes oxygen from Vox N2 tank (whoops)
-Message when brain is EMP'd
-Brain icon
-Better tank icon
-Vox can actually talk now

Ziiro commented on code/modules/mob/living/carbon/human/species_types/vox.dm:
```diff
@@ -0,0 +1,46 @@
+	C.dna.species.limbs_id = vox_body_of_choice.limbs_id
+	var/vox_tail = C.dna.features["vox_tail"]
+	var/datum/sprite_accessory/vox_tails/vox_tail_input = GLOB.vox_tails_list[vox_tail]
+	vox_tail_input = vox_body_of_choice.limbs_id
```
So this line totally doesn't work, but what I want it to do is pull the limbs_id of the body and then choose the tail based on that.

Ziiro commented on code/modules/mob/living/carbon/human/species_types/vox.dm:
```diff
@@ -0,0 +1,46 @@
+	vox_tail_input = vox_body_of_choice.limbs_id
+	var/vox_eyes = C.dna.features["vox_eyes"]
+	var/datum/sprite_accessory/vox_eyes/vox_eyes_input = GLOB.vox_eyes_list[vox_eyes]
+	vox_eyes_input = vox_body_of_choice.eye_type
```
Same as tails, except I've made a var specific for vox_bodies that are `eye_type`, and there's `two_eyes` and `three_eyes`

The two eyes for the main set of bodies. Three eyes for the Auroras. If I can just make it read them correctly..

Ziiro committed:
integrates species clothing (#387)
Also fixes up default/random features code.

Ziiro commented:
One oddity I'll have to fix later. When attempting to wear a piece of clothing with restricted species:

![dreamseeker_2017-12-31_09-28-19](https://user-images.githubusercontent.com/26494679/34463040-333246fa-ee0e-11e7-9d80-292814a93e1c.png)

Spartan6 commented:
You remove the two species thing... a nice thing...

And keep the normal talking?

*scream

Ziiro commented:
I mean, that's up to the player. Trying to enforce vox-speak via message changing on the code level would be difficult.

¯\\_(ツ)_/¯

Ziiro committed:
STOP WHINING TRAVIS

Ziiro committed:
SHITBIRDS WERE A MISTAKE
-Tails work
-Eyes work
-Quills and Facial Quills lightened up for color
-Clothing groundwork has been put in

Ziiro commented:
Alright so the initial set of clothing options are in.
![dreamseeker_2018-01-01_22-38-41](https://user-images.githubusercontent.com/26494679/34475183-c263854e-ef46-11e7-8c38-8c66db72680b.png)
(Ignore the CMO looking weird, I'm pretty sure their eye color ended up the same as their body color)

There is way, *way* too much for me to test everything and fix it by hand, however. So there will be problems with this when we get around to playtesting it.

Ziiro committed:
skrek
Adds a number of missing hats/items/etc
Adds oxymod
Preps systems for body markings

Ziiro committed:
More customization, sprite fixes
-Tail markings
-Body markings
-A lot of clothing adjustments/restrictions
-Vox breath mask sprite changes

Ziiro commented:
![2018-01-02_12-29-29](https://user-images.githubusercontent.com/26494679/34511519-6d563c1c-f019-11e7-8616-7b17893d6f7c.gif)

Body and tail markings are done. Doesn't look too bad. This makes all of the visual parts done, I think.

To-do list updated a post back. Also Flattest is working on an RGB Blood system that will make it so vox can have proper blue blood, as well.

Ziiro committed:
NEVER IS GONNA GIVES YOUS UPS

Ziiro commented:
YOU WISH

Ziiro committed:
NEVER GONNA LETS YOUS DOWNS

Ziiro commented:
Remaining known issues:
* Overlays
* Toxin damage from breathing oxygen is not nearly high enough
* Tail doesn't update when husking, but I don't see a way to fix that that doesn't fuck everything up worse.

FlattestGuitar commented on code/game/objects/items/tanks/tank_types.dm:
```diff
@@ -181,3 +179,26 @@
+
+/obj/item/tank/internals/emergency_oxygen/vox
+	name = "vox n2 tank"
+	desc = "A vox nitrogen tank made of a strong, light weight alloy; this tank is capable of holding gasses at high pressures without exploding. Due to it's light weight, don't expect to robust any dustlungs with it, skrek."
```
its*

FlattestGuitar commented on code/game/objects/items/tanks/tank_types.dm:
```diff
@@ -181,3 +179,26 @@
+	force = 0 // No, you don't get to break out of the brig using your voxygen tank, shitbird.
+	distribute_pressure = TANK_DEFAULT_RELEASE_PRESSURE
+	volume = 8 // Larger than a standard emergency tank, smaller than a double.
+	attack_verb = list("shitbirded")
```
too OOC. Make it things like "skrekked" or "skreed" or something.

Ziiro replied:
Skrekked is for the tongue. I'll figure it out, though. Hmm.

imsxz replied:
is it really too OOC? i call em shitbirds ICly

FlattestGuitar commented on code/modules/clothing/masks/breath.dm:
```diff
@@ -36,3 +36,12 @@
 	permeability_coefficient = 0.01
 	equip_delay_other = 10
+
+/obj/item/clothing/mask/breath/vox
```
I'd remove the mask. No point in it being a separate thing if it already molds itself to fit other species. Just use a regular one on Vox.

Ziiro replied:
iirc it has to be a unique thing because of the `gas_transfer_coefficient = 0.01`, which is not on the base breath mask.

FlattestGuitar replied:
That variable literally does nothing.

FlattestGuitar commented on code/modules/mob/living/brain/brain_item.dm:
```diff
@@ -147,6 +147,23 @@
+/obj/item/organ/brain/cybernetic/vox
+	name = "Vox cortical stack"
+	slot = "brain"
+	desc = "A vox cortical stack - considered to be their brain. Its existance is the center of debate."
```
a) existence*
b) what does that even mean? People can see it pretty clearly, right? What's there to debate?

Ziiro replied:
A joke about vox not having brains, mostly.

FlattestGuitar commented on code/modules/mob/living/brain/brain_item.dm:
```diff
@@ -147,6 +147,23 @@
+	origin_tech = "biotech=3"
+
+/obj/item/organ/brain/cybernetic/vox/emp_act(severity)
+	to_chat(owner, "<span class='warning'>Your cortical stack has been damaged!</span>")
```
Make it more vague. "Your head hurts...", for example. Bonus points if the message is the same thing as something completely different somewhere in the code.

imsxz replied:
oof ouch my cortical stack

FlattestGuitar commented on code/modules/mob/living/brain/brain_item.dm:
```diff
@@ -147,6 +147,23 @@
+	to_chat(owner, "<span class='warning'>Your cortical stack has been damaged!</span>")
+	switch(severity)
+		if(1)
+			owner.adjustBrainLoss(50)
```
I'd put more pressure on a stun, not brain damage.

Something like a 10 second knockdown and `rand(0, 10)` brainloss.

FlattestGuitar reviewed (dismissed):
Not a complete review, but most things look fine so far.

imsxz commented on code/modules/mob/living/carbon/human/species_types/vox.dm:
```diff
@@ -0,0 +1,53 @@
+
+	while(i<=sounds)
+		i++
+		newname += pick("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah")
```
"ya" listed twice. perhaps replace one with "you"?

imsxz replied:
meant replace with "yi" fukken autocorrect

imsxz reviewed (commented):
Kiki yaya

Ziiro committed:
NEVER IS GONNAS RUNS ROUNDS AND DESSERTS YOUS
various fixes to brain item, balance concerns, organs, tank.

Ziiro committed:
NEVER GOINGS MAKES YOUS CRYS
Collapsed vox breath mask into standard breath mask
Removed vox skeleton

Ziiro commented:
Required for completion:
* Damage overlays
* #418 integration
* Toxin damage from breathing oxygen is not nearly high enough
* Tail doesn't update when husking, but I don't see a way to fix that that doesn't fuck everything up worse.

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

Ziiro committed:
NEVER GOINGS SAYS GOODBYES
Blood merge/update
Overlays

Ziiro committed:
NEVER GOINGS TELLS LIES AND HURTINGS YOUS
-Makes toxic damage for breathing oxygen actually meaningful.

Ziiro commented:
Outstanding known issues:
* Tail doesn't update when a vox is husked.

Ziiro committed:
Minor stuff
-Shitcode tail husking (not 100%)
-Removes oxymod, because it's not used

AndrewMontagne commented:
Using Select Equipment on a vox does not give them a N2 tank and mask

Ziiro commented:
Shaft Miner vox spawn with.. nothing

Ziiro commented:
* Waistcoats
* Jackets

Ziiro commented:
* Vox language needs an icon

SamHPurp commented:
helmets aren't showing up, particularly the engineering helmet I tried just now.

Ziiro commented:
* Pimp Hat

Ziiro commented:
Bug/issues:
* Select Equipment does not give vox breath masks or voxygen tanks
* Vox language needs an icon
* Tail husking is shitcode
* Vox can't eat without removing their mask, **_and it's very bad for them to do so_**. I'm not sure what to do here.

Clothing to be fixed or locked out:
* Bee Keeping suit/hat
* Witchhunter
* Holiday Priest
* Chaplain Robes with hood
* Student Robe
* Pimp Hat
* Explorer's hood / Explorer's Suit / Explorer's gas mask ((HIGH PRIORITY))
* Shaft Miner's Jumpsuit (Lavaland)
* Sombrero
* Engineering Hardhats (Firefighter hardhat?)
* Waistcoat
* Maid apron
* Jackets? (Bomber, military I assume?)

EldritchSigma commented:
Ultra high priority: Check if vox can wear bikinis.

Ziiro committed:
progressings yaya
-A handfull of clothing fixes/locks
-Fixes "select equipment" (mostly)

Ziiro committed:
Shitbirds are now lavaland ready
ENJOY THE SHITTY CODER SPRITES

Ziiro commented:
![dreamseeker_2018-01-22_22-51-25](https://user-images.githubusercontent.com/26494679/35260137-fca3e90c-ffc6-11e7-8a3a-55c3b291ea3d.png)

Small issue.
Their base head eyes appear when it does `HIDEFACE` when you enable the hood, because that hides their recolored eye feature:

![2018-01-22_22-56-51](https://user-images.githubusercontent.com/26494679/35260232-9f2efa86-ffc7-11e7-8450-d58d4b4c5a79.gif)

I could just make those blue eyes go away, I guess, and blend them into the head?
¯\\_(ツ)_/¯

Ziiro commented on code/modules/mob/living/carbon/human/species_types/vox.dm:
```diff
@@ -0,0 +1,58 @@
+
+/datum/species/vox/spec_life(mob/living/carbon/human/H)
+	if(H.disabilities & HUSK)
+		H.dna.features["vox_tail"] = "voxhusk"
```
I'd prefer a better way to do this. Christ this is bad.

Ziiro commented on code/modules/surgery/organs/lungs.dm:
```diff
@@ -316,6 +317,16 @@
+	safe_oxygen_max = 1 //This is toxic to us
+	safe_nitro_min = 16 //We breathe THIS!
+	oxy_damage_type = TOX //Oxygen poisons us
+	oxy_damage_multiplier = 10 // BADLY.
```
Base level is 1/second, roughly. This increases it to.. 10. I might kick it down to 5, that way it's not as bad, but it still needs to be bad.

Ziiro replied:
Actually I will make it 5. That way they can eat without fucking dying. 🤔

Ziiro commented on code/modules/admin/verbs/debug.dm:
```diff
@@ -664,9 +664,10 @@ GLOBAL_PROTECT(LastAdminCalledProc)
 		else
 			M.equipOutfit(dresscode)
-
+			M.dna.species.after_equip_job(src, M)
```
This was a pretty quick fix, but I'm sure I'm overlooking something bad that could come from this.

I have to alter a few loadouts, because some of them are a wee bit broken - the detective spawning with a cigarette stops a mask from being added. Leading to the death of detective voxxy.

Ziiro commented:
Additional note to self:
* Finish adjusting the explorer satchel, it looks like shit

Birdtalon commented:
Yaya!

SamHPurp commented:
So they are currently still `spaceimmune`?

Slignerd commented:
👀

Slignerd commented:
👀

Ziiro commented:
@SamHPurp I have answered you on that at least **twice in the last day**, with explanation you have apparently ignored.

Also if you look at the code, it's called `RESISTPRESSURE`. That allows them to resist moderate amounts of pressure (but super extremes still crush them)

So let's try it again.

They are space immune, BECAUSE:
* Hardsuits are **HARD**suits. The idea of them transforming to fit a species is dumb. Additionally, /tg/ has a few hardsuits that vox do not have, or they use updated sprites. Also why would a human corporate have xeno-transforming suits?
* IN ORDER FOR VOX TO DO JOBS EFFECTIVELY, they need to be able to spacewalk. Without the ability to spacewalk, no: Engineer, Paramedic or Security vox allowed. Additionally, since they can't be command, that _really_ limits potential work.
* In theory, I could remove `RESISTPRESSURE` and `RESISTCOLD` and make a few vox-only hardsuits. However this would take map modifications to add the hardsuits, or a machine that can transform the hardsuits from one to another. Additionally, there would be a few they just couldn't wear, because the graphics on some aren't up to standards.

SamHPurp commented:
No need to get shitty my dude.

I strongly feel, as I've said from the beginning, that spaceproof should absolutely not be a quality any round-start races possess.

Its impossible to balance around, and is even the reason why Plasmamen were changed to be encased in a jumpsuit, not incased in a space suit - so even Plasmas aren't spaceproof.

Ziiro commented:
Okay. I understand that. So what solution do you want me to do then, so I don't waste time on unacceptable solutions?

FlattestGuitar commented:
We need to get our shit together and talk this through from the design standpoint.

I think I agree with Purpose on them not being spaceproof. It's not /that/ much work to redo a few hardsuits, I can help with that much myself.

Ziiro committed:
NOES!! DEVELOPINGS TAKING SPACEPROOFINGS!!! *AFLAP
-Vox no longer space proof
-Can wear most common hardsuits now

Ziiro commented:
Poof. It's done.

However, balance wise they kind of suck now. Also I feel like this is a step backwards for "unique experience", but oh well:

Pros:
* Yaya!
* 20% Less burn damage

Cons:
* Can't be cloned
* Stunned for 20% longer
* 2x tox damage
* 10 damage a second when exposed to oxygen.

Additional notes:
All of the clothing DMIs are a mess. I have to prune them for things that don't exist in this codebase.

PromethTK11 commented:
Is nice to see people using my sprites <3

FantasticFwoosh commented:
> @Slignerd : This is a bad idea.

We've already been over this.

Personally i think Paradise's ideas on vox only clothing are kind of screwy but that's sort of a problem with the anatomy of the race. Every suit will eventually demand a vox variant of a suit when many intermediate external suits like labcoats can fit fine.

- There are so many diehard vox supporters that limit the expression by insistence on being on standard with high RP servers.

I have alternative parrot-people sprites though if you want to do something else.

![](https://puu.sh/yZI3V/fd119dc8e5.jpg)

Ziiro commented:
>Every suit will eventually demand a vox variant of a suit when many intermediate external suits like labcoats can fit fine.

I don't agree with this. If the spriter/coder/whoever doesn't actually want to deal with it, they can just slap the `species_restricted = list("exclude","Vox Outcast")` on there, and then vox can't wear it. If a shitbird player wants to wear something that doesn't have a sprite, they can sprite the thing for themselves.

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

# Conflicts:
#	code/modules/client/preferences.dm
#	config/game_options.txt

Ziiro committed:
blurb for description

Ziiro commented:
Vox are now SPACE RESISTANT:
* Without chems or medicine or anything else. A vox can spacewalk for about two minutes before slipping into hardcrit.

I'd like to apologize to @SamHPurp and @FlattestGuitar because I am retarded. Let us never speak of this again.

Ziiro committed:
YAYA IS GOODS
-RESISTPRESSURE is back.
-Fixes their eye layering with clothes
-Lowers threshold for breathing
-Removes NOSCAN, but adds NOCLONE disability that can be overcome with parts

Ziiro committed:
VOX: NOW WITH 200% MORE SHRIEKING

Ziiro committed:
Vox Organ changes
-All applicable vox organs are now ORGAN_ROBOTIC
-Some have very minor EMP effects
-None of them have icons yet (working on it)

chesse20 commented:
why are vox uncloneable? Idk about vox on paradise but i play vox all the time on /vg/ and the balance is fine with them being clonable round start. I mean they can't be head roles and move really slowly because they are usually wearing the hardsuits they spawn in and have to wear a nitrogen tank at all times. Making them unclonable round start seems like overkill

Ziiro commented:
To-do:
* Vox Pidgin has no icon (Needed: 32x32 icon that's a.. beak? a claw? Not sure)
* Vox organs have no icons (Working on it.)
* Can't get husking of the tail to work outside of `spec_life` ( ??????? )
* Map changes will probably be needed to add some spare vox tanks to medbay. (Future PR)
* If revived from the dead, the "choking, not enough n2" alert doesn't disappear.

>why are vox uncloneable? Idk about vox on paradise but i play vox all the time on /vg/ and the balance is fine with them being clonable round start. I mean they can't be head roles and move really slowly because they are usually wearing the hardsuits they spawn in and have to wear a nitrogen tank at all times. Making them unclonable round start seems like overkill

Vox on para and vg are different, I think. I don't know about /vg/ vox, but para vox are **completely spaceproof**. That's a pretty huge deal, so making them "hard mode" the sense that they can't be cloned kind of makes sense.

Since I've removed the **complete** space-proof-ness, it is logical to remove their **complete** inability to be cloned. But balance is a constant battle, so we'll see what makes it to live.

For the moment:
Pro:
* Take 20% less burn damage
* Breathe from internals at all times, making them more resistant to viruses
* Can spacewalk for roughly 60% (I think?) longer than any other organic.

Cons:
* 2x tox damage taken (Not 100% on this, might adjust down.)
* Oxygen is VERY POISONOUS (6 tox/sec, but I might adjust this downwards)
* Cloner needs to be a certain level to clone them (Also the process of doing so is hard. You have to mask them up and get the tank on them quickly when they're ejected from the pod)
* A fair number of clothing is off limits to them for sprite reasons
* Organs have minor EMP effects (Brain takes brain damage)

Ziiro committed:
Adjusts toxin damage for breathing o2

Ziiro committed:
spare vox tanks now in cargo "internals" crate.

Ziiro committed:
vox organs, icons

AndrewMontagne committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

Ziiro committed:
adds vox-pidgin icon

Ziiro commented:
Organ and Vox Pidgin icons brought to you by xantholne

Ziiro committed:
Loadout fixes, husking fixes

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

Ziiro commented:
* Need a xenobio slime potion

Xantholne commented:
- Does not have a sprite for treasure hunter hat, jacket, or uniform

_**IS BELONGS IN A MUSEUMS!**_

Ziiro commented:
>Does not have a sprite for treasure hunter hat, jacket, or uniform

When you sprite it™
>Vox can't *wag

Fuck no
>Vox Can't *quill

They make enough noise. Pass.

Ziiro committed:
vox xenobio potion

Ziiro committed:
changes xenobio recipe

Xantholne commented:
Zero is just a rude dude who doesn't want vox to ruffle their quills or wag their raptor tails

Ziiro commented:
* NEEDS CHITINOUS MASS SPRITE
* NEEDS FLESH MASS SPRITE

SamHPurp commented:
Those organs are awful. The blue I can dig, but the animation is bad. It looks ugly and it also gives the impression they are functional, when infact they are lying on the floor.

The racial timer for Vox to be able to be played with be `100h` as crew.

Questions:
- Can Vox eat through their mask?
- Do the equipment admin-stuff equip N2 to vox? (If not, this is something for a future PR, you won't need to do it for this one. Plasmamen should be included in this future PR)

Ziiro commented:
>Can Vox eat through their mask?

No. Not sure how to square this without fucking with the code that honestly doesn't need fucking with. However the lift mask *does* work for them, and they can eat, then inject some charcoal or get into a sleeper.

>Do the equipment admin-stuff equip N2 to vox? (If not, this is something for a future PR, you won't need to do it for this one. Plasmamen should be included in this future PR)

It's underneath `equip_after_job`, and I actually put that into that command, so yes.

SamHPurp commented:
Fandabbydozey.

Another playtest would be good to catch more DMI / equipment problems.

Ziiro committed:
removes christmas lights from vox organs (except for brain)
adds sprites to cling armor/space suits, bodies are not done yet.

Ziiro committed:
blocks cult items without sprites

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

# Conflicts:
#	code/modules/clothing/suits/jobs.dm

Ziiro committed:
implements Vivi's cling armor sprites

Ziiro committed:
s w o l e  b i r d
ling space suit for vox

Xantholne commented:
@Ziiro Clown and mime mask have an exception for food and drink, maybe check out what they have that lets it go through.

ike709 commented:
I've given up on putting [READY] at the front of my PRs because I know it'll still take a week for a review.

Ziiro commented:
@Xantholne 
Breath masks have `MASKCOVERSMOUTH`, Clown masks only have `MASKCOVERSEYES`

I could remove the COVERSMOUTH, but ehhhhhhhhhhhhhhhhh

Xantholne commented:
I'd set it to covereyes just for vox sanity.

But at the same time tbqh I don't give a shit about shitbirbs taking tox damage from needing to eat. It's funny.

Cactus2388 commented:
Damage from eating is a FEATURE

Ziiro commented:
@Xantholne Unfortunately I've made the vox breath mask the generic breath mask. If I were to do that, everyone could eat while wearing a breath mask. Which is ⛔️

FlattestGuitar commented on code/modules/reagents/chemistry/recipes/others.dm:
```diff
@@ -181,6 +181,11 @@
 	results = list("golemmutationtoxin" = 1)
 	required_reagents = list("unstablemutationtoxin" = 1, "silver" = 1) //not too hard to get but also not just there in xenobio
 
+/datum/chemical_reaction/vox_mutation_toxin
```
Personally, I think we should take a look at these things and kick quite a few of them out. Turning someone into a Plasmaperson or a Vox out of sudden will kill them really quickly, and I don't think we should hand people an easy way to do that, especially when going back on it is so hard.

Ziiro replied:
I think if someone spends enough time to get the slimes and make these sorts of things, then it's a decent tool as a traitor. They're using it on others as a non-antag, then that's some pretty clear self-antagging and worthy of punishment.

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

# Conflicts:
#	code/modules/reagents/reagent_containers/blood_pack.dm

Ziiro committed:
fixes whitespace problems from merge conflict resolution

Ziiro committed:
oops

Ziiro committed:
some minor sprite and graphics tweaks

Ziiro committed:
Merge branch 'master' into UnfortunatelyAddsShitbirds

# Conflicts:
#	code/modules/surgery/organs/liver.dm

Ziiro commented:
Ran a testmerge for two shifts, no major issues reported.

Ziiro committed:
OOPS

AndrewMontagne reviewed (dismissed):
is goods to merges yaya

Ziiro committed:
vox can now be clinged (but there are issues with this)

Ziiro commented:
So vox can now be clinged, however

![dreamseeker_2018-03-08_20-14-28](https://user-images.githubusercontent.com/26494679/37189255-9193f220-2310-11e8-9819-9651d60f22cc.png)

It's not copying the body feature in the process.

🤔 

Even with this `small` bug they're still playable

Ziiro committed:
tweaks stun from 20% more to simply 10% more

AndrewMontagne reviewed (approved):
LGTM

Ziiro commented:
YAYA
